updating/updating-restricted-network-cluster/restricted-network-update.adoc

```diff
@@ -6,6 +6,8 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
+Use the following procedures to update a cluster in a disconnected environment if you do not have access to the OpenShift Update Service in the disconnected environment.
```

Comment:
Also I am wondering if you could simplify this sentence a bit. As it reads, it is a bit of a mouthful. I am not sure if it changes the meaning to do something like:
```suggestion
Use the following procedures if you do not have access to the OpenShift Update Service in the disconnected environment.
```
This sentence would imply that you are trying to update a cluster in a disconnected environment (_I think_).

Reply:
I agree that it's an awkward sentence, though I'm a little wary about an alternative that doesn't explicitly mention updating a cluster. How about the following option:

"Use the following procedures to update a cluster in a disconnected environment without access to the OpenShift Update Service."

Still not the greatest, but I think it's tough when you need to mention 1. updating a cluster 2. the fact that it's a disconnected cluster and 3. the fact that OSUS is not available in that environment

Reply:
Yeah I like your suggestion. It reads easier imo